rero_ils/modules/operation_logs/listener.py

```diff
@@ -50,6 +50,18 @@ def operation_log_record_update(sender, record=None, *args, **kwargs):
     build_operation_log_record(record=record, operation='update')
 
 
+def operation_log_record_delete(sender, record=None, *args, **kwargs):
+    """Create an operation log entry after record deletion.
+
+    Checks if the record is configured to keep logs of its deletion.
+    If enabled, a new operation log entry will be added. Method is called after
+    record deletion by connecting to signals'after_record_delete'.
```

Comment:
```suggestion
    record deletion by connecting to signals 'after_record_delete'.
```
Is it better like this ? :arrow_down: 
```
   If enabled, a new operation log entry will be added. Method is called after
    record deletion by connecting to `after_record_delete` signal.
```